src/engine/SCons/Tool/msvs.py

```diff
@@ -75,6 +76,10 @@ def xmlify(s):
 def processIncludes(includes, env, target, source):
     return SCons.PathList.PathList(includes).subst_path(env, target, source)
 
+# Convert a file to its absolute path.
+def processSource(file):
+    return SCons.Script.File(file).abspath
```

Comment:
Shouldn't this be SCons.Node.FS.File and not SCript...

Reply:
Not sure. Do you know why SCons.Script exposes things like File and Dir in https://github.com/SCons/scons/blob/master/src/engine/SCons/Script/__init__.py#L323 ?

Reply:
So you can use File,etc in SConscripts..